dipy/tracking/utils.py

```diff
@@ -1014,3 +1019,60 @@ def _as_segments(streamline, break_points):
         yield seg
     for seg in _part_segments(streamline[::-1], break_points[::-1]):
         yield seg
+
+
+def max_angle_from_curvature(min_radius_curvature, step_size):
+    """Get the maximum deviation angle from the minimum radius curvature.
+
+    Parameters
+    ----------
+    min_radius_curvature: float
+        Minimum radius of curvature in mm.
+    step_size: float
+        The tracking step size in mm.
+
+    Return
+    ------
+    max_angle: float
+        The maximum deviation angle in radian,
+        given the radius curvature and the step size.
+
+    References
+    ----------
+    For more information:
+    https://onlinelibrary.wiley.com/doi/full/10.1002/ima.22005
+
+    """
+    max_angle = 2. * np.arcsin(step_size / (2. * min_radius_curvature))
+    if np.isnan(max_angle) or max_angle > np.pi / 2 or max_angle <= 0:
+        max_angle = np.pi / 2.0
```

Comment:
If the max_angle will default to pi/2, it might be a good idea to display a message about this. Otherwise, there will be a mismatch between min_radius_curvature and angle, which might be difficult to notice later on.

Reply:
A warning? Maybe a logging message? Please don't put any more "print" calls in the code-base.

Reply:
I used a warning